### DISCUSSION TITLE
Needs to be refiled

### DISCUSSION THREAD

**Discussion by Haus-Master** [Courseplay/Courseplay_FS22#3063] [Q&A]
Hey, is it possible for the worker to stay in the vehicle being used for seeding, spraying, spreading, etc whilst it waits to be refilled then carry on once the player has refilled the implement?

I'm having to re-enter the vehicle once I have filled the implement and continue the course.
is it possible for example to be spreading Lime, the spreader runs out of lime, the worker then stops, I refill the spreader, then the worker carries on the course from it's current point until completion or it need refilling again?

Thanks.

**Comment by Tensuko** [comment]
Right now it is not possible.
I will write that down.

**Reply by Haus-Master** [reply]
Okay,  thank you for the reply and taking note of it.

Also side note thank you for fixing the spot spraying issue.
